api/v1alpha1/bgppeer_webhook.go

```diff
@@ -89,6 +92,25 @@ func (bgpPeer *BGPPeer) validateBGPPeer(existingBGPPeersList *BGPPeerList) error
 	return err
 }
 
+func (bgpPeer *BGPPeer) validateBGPPeersKeepaliveTime(existingBGPPeersList *BGPPeerList) *field.Error {
+	holdTime := bgpPeer.Spec.HoldTime
+	keepaliveTime := bgpPeer.Spec.KeepaliveTime
+
+	// Keepalivetime is not set then return w/o doing any validation
+	if keepaliveTime == 0 {
+		return nil
+	}
+	if holdTime == 0 {
```

Comment:
Another comment here. This may be unexpected, but makes sense because if we get here, keepalive is > 0